crypto/BRCryptoWalletManagerClient.c

```diff
@@ -698,7 +666,7 @@ cwmTransactionEventAsBTC (BRWalletManagerClientContext context,
 
             BRCryptoTransferState oldState = cryptoTransferGetState (transfer);
 
-            if (CRYPTO_TRANSFER_STATE_INCLUDED == oldState.type &&
+            if (CRYPTO_TRANSFER_STATE_SUBMITTED != oldState.type &&
                 (0 == event.u.updated.timestamp || TX_UNCONFIRMED == event.u.updated.blockHeight)) {
                 // The transfer is not included so set it to the submitted state at this point
                 BRCryptoTransferState newState = cryptoTransferStateInit (CRYPTO_TRANSFER_STATE_SUBMITTED);
```

Comment:
Might be the comment 'The transfer is not included ...' is inconsistent with the logic.  Maybe a general explanation in front of the `if` - as in "if the transfer is not submitted and is missing values for timestamp and blockHeight, then return it to submitted" ?